common/Scripts/helpers.sh

```diff
@@ -1,30 +1,49 @@
 #!/usr/bin/env bash
 
+# shellcheck wants to add '|| exit 1' willy nilly, so let's
+# build a nice 'failmsg' function instead.
+function failmsg() {
+    echo "$*"
+    return 1
+}
+
 # Goes to the root directory of the solus packages
 # git repository from anywhere on the filesystem.
 # This function will only work if this script is sourced
 # by your bash shell.
 function gotosoluspkgs() {
-    cd "$(dirname "$(readlink "${BASH_SOURCE[0]}")")/../../" || exit 1
+    cd "$(dirname "$(readlink "${BASH_SOURCE[0]}")")/../../" || \
+        failmsg "Unable to find the solus packages git dir."
 }
 
 # Goes to the root directory of the git repository
 function goroot() {
-    cd "$(git rev-parse --show-toplevel)" || exit 1
+    local pkgroot="$(git rev-parse --show-toplevel)"
```

Comment:
```
In helpers.sh line 21:
    local pkgroot="$(git rev-parse --show-toplevel)"
          ^-----^ SC2155 (warning): Declare and assign separately to avoid masking return values.
```
https://www.shellcheck.net/wiki/SC2155